x-pack/plugins/apm/public/components/app/service_overview/service_overview_errors_table/index.tsx

```diff
@@ -30,9 +30,9 @@ interface Props {
   serviceName: string;
 }
 type ErrorGroupMainStatistics =
-  APIReturnType<'GET /internal/apm/services/{serviceName}/error_groups/main_statistics'>;
+  APIReturnType<'GET /internal/apm/services/{serviceName}/errors/main_statistics'>;
 type ErrorGroupDetailedStatistics =
-  APIReturnType<'GET /internal/apm/services/{serviceName}/error_groups/detailed_statistics'>;
+  APIReturnType<'GET /internal/apm/services/{serviceName}/errors/detailed_statistics'>;
```

Comment:
Maybe we should rename the variables removing the `Group` from it too?!

Reply:
If I rename these, then for consistency I will need to rename errorGroups variables everywhere, I rename these endpoints to make them consistent with how we write the paths for the errors page `x-pack/plugins/apm/server/routes/errors/route.ts`